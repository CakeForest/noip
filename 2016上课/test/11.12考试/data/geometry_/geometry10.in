100
541 308 981 132 120 674 852 52 680 805 725 850 334 80 895 938 396 550 813 148 610 712 350 608 978 0 264 828 481 938 417 680 878 567 992 570 410 552 740 104 888 844 984 905 402 940 840 574 895 804 56 676 950 959 648 528 88 800 779 434 742 668 989 780 371 240 585 28 868 700 890 0 961 520 358 304 318 496 780 841 8 502 548 590 6 245 35 600 878 740 150 167 444 620 490 12 735 735 612 976 
544 40 460 955 896 442 738 544 712 840 620 496 546 169 210 96 250 310 476 647 692 221 675 469 418 304 415 60 199 936 425 5 783 531 600 250 91 420 625 520 120 460 950 146 558 832 660 396 760 834 652 21 240 105 650 385 128 804 424 400 250 390 430 36 433 396 500 920 640 732 810 816 936 180 146 72 560 825 818 955 174 210 104 400 319 0 445 658 819 336 930 160 65 78 914 314 562 372 654 360 
100
943 330
562 538
270 646
227 808
147 608
100 315
874 460
524 249
748 128
120 460
584 916
222 976
324 982
20 344
622 315
969 660
200 32
34 446
892 60
200 220
693 204
324 328
373 684
11 125
896 46
625 974
290 106
600 272
608 152
316 438
384 395
853 42
621 416
285 621
868 695
574 50
305 359
0 195
752 308
239 604
855 384
168 395
232 172
680 758
853 240
504 145
695 656
744 535
412 800
296 562
732 620
267 442
581 766
23 121
942 656
106 323
360 902
302 560
670 20
900 835
184 672
38 376
388 375
830 928
472 60
165 50
667 802
544 296
408 653
896 748
975 412
945 808
72 764
518 652
606 451
782 377
100 178
932 222
947 848
24 392
762 672
912 560
128 750
45 896
808 50
418 723
516 556
788 115
160 925
237 328
935 80
624 473
431 122
328 155
6 191
996 593
835 542
199 873
465 775
439 296
