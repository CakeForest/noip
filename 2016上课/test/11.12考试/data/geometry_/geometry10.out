100
100
92
100
79
38
100
75
84
62
100
100
100
34
92
100
19
54
92
35
88
65
100
15
91
100
32
87
70
77
78
87
100
91
100
53
66
21
100
87
100
58
33
100
100
59
100
100
100
88
100
74
100
15
100
42
100
88
59
100
90
42
77
100
42
18
100
81
100
100
100
100
88
100
100
100
26
100
100
42
100
100
91
97
80
100
100
88
100
57
100
100
49
39
21
100
100
100
100
72
